AirCasting/Settings/BackendURLValidator.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  BackendURLValidator.swift
+//  URLValidation
+//
+//  Created by Pawel Gil on 25/06/2021.
+//
+
+import Foundation
+
+class BackendURLBuilder {
```

Comment:
I think we can make it `final`. Or perhaps even make it a `struct` as this includes behavior only